frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -118,10 +109,39 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
     const usableConfigs = globalConfigs
       .filter((item) => this.checkFlags(item))
       .map((item) => item.properties);
-    const allItems = usableConfigs.length > 0 && items.concat(usableConfigs);
-    const sortedItems =
-      usableConfigs.length > 0 ? _.sortBy(_.flatten(allItems), 'kind', 'asc') : items;
-
+    let allItems = items.concat(usableConfigs);
+    allItems = allItems
+      .map((item) => {
+        const apiExplorerLink = `/api-resource/cluster/${referenceForModel(item.model)}`;
+        const resourceLink = resourcePathFromModel(item.model, item.name, item.namespace);
+        return {
+          ...item,
```

Comment:
Don't pass more than is necessary. You should be able to remove this line. I would add a `label` prop to the row items since `kind` is not accurate when it's not a k8s resource.

```ts
label: item.kind,
id: item.uid,
path: resourceLink,
```